styleSign.css

```diff
@@ -0,0 +1,207 @@
+body {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  font-family: 'Poppins';
+  background-color: #F9FAFF;
+}
+
+.main-wrapper {
+  background-color: #F9FAFF;
+}
+
+.image-background {
+  width: 100%;
+  z-index: -1;
+}
+
+.image-background__top {
+  width: 100%;
+  margin-bottom: -50%;
+}
+
+.form-signup {
+  background-color: #F9FAFF;
+  border-radius: 24px 24px 0 0;
+  width: 100%;
+  z-index: 10;
+  -webkit-transform: translateY(-5%);
+          transform: translateY(-5%);
+}
+
+.form-signup_move {
+  -webkit-transform: translateY(-75%);
+          transform: translateY(-75%);
+}
+
+.name-page {
+  margin: 5% auto;
+  width: 80%;
+  color: #202225;
+  padding-top: 10%;
+  font-weight: 600;
+}
+
+.wrapper-input {
+  width: 80%;
+  margin: 5% auto;
+  border: 1px solid #DCE0EC;
```

Comment:
save all ur colors in variables
```
`:root {
  --main-bg: #f9faff;
  --title-color: #202225;
  --text-color: #a1abc9;
  --input-color: #dce0ec;
  --input-accent: #7297ff;
  --input-error: #ff2567;
  --white-color: #ffffff;
  --online-color: #34c197;
  --doctor-bg: #e3ebff;
}`
```
.wrapper-input {
....
`border: 1px solid var(--input-color)`
}